README.md

```diff
@@ -77,29 +77,192 @@ The authentication flow is as follows:
+
+#### Install rbenv
+
+On MacOS and Linux, for managing multiple versions of Ruby (needed for _Fastlane_ and _CocoaPods_), we recommend the use of [rbenv](https://github.com/rbenv/rbenv).
 
 The Ruby version used in this project is stored in [.ruby-version](.ruby-version).
 
 If you already have rbenv installed and configured on your system, the correct Ruby version will be set, when you access the app directory.
 
-Some dependencies (eg CocoaPods) are installed via [bundler](https://bundler.io/).
+To install, follow the steps described below.
+
+##### Installing `rbenv` on MacOS
+
+You should already have installed `brew` so use:
+
+```
+brew install rbenv
+```
+
+Brew installs `rbenv` in the path so no more steps are needed.
+
+##### Installing `rbenv` on Linux 
+
+This is the generic installation procedure for Linux that should work on many distributions. The procedure has been tested on Ubuntu Linux. 
+
+```
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+mkdir ~/.rbenv/plugins
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+```
+
+Add `rbenv` to the PATH as follows then reload the intialization file
+
+```
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >>$HOME/.bashrc
+source $HOME/.bashrc
+```
+
+Verify you have installed it correctly with the command `which rbenv`.
+
+#### Completing and verifying configuration
+
+Either on Mac or Linux you need to add to your shell the initialization command thjen reload the configuration:
+
+```
+echo 'eval "$(rbenv init -)"' >>$HOME/.bashrc
+source $HOME/.bashrc
+```
+
+(if you use a different shell than bash you may need to adapt the command to your shell initialization files).
+
+
+Before you can install your version of Ruby, you need a C compiler and some libraries. On Ubuntu or Debian based systems use:
+
+```
+sudo apt-get install build-essential libssl-dev libreadline-dev zlib1g-dev
+ ```
+
+Now you can install your version of `ruby` using `rbenv` (replace `<work-dir>` with your actual work directory)
+
+```
+cd <work-dir>/io-app
+rbenv install
+```
+
+You should verify that the output of the `rbenv version` command and the content of the file `.ruby-version` are the same:
+
+For example (replace `<work-dir>` with your actual work directory):
+
+```
+$ cd <work-dir>/io-app
+$ rbenv version
+2.4.2 (set by <work-dir>/io-app/.ruby-version)
+$ ruby -v
+ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-linux]
+$ cat .ruby-version
+2.4.2
+```
+
+#### Install bundler and cocoa
+
+Some dependencies are installed via [bundler](https://bundler.io/) and [cocoapods](https://cocoapods.org/) 
+
+Note that on Linux you do not need CocoaPods as you can only build for Android.
+
+They are  Ruby application and it is recommended you install both using the ruby managed by rbenv, with this procedure:
+
+```
+cd <work-dir>/io-app
+rbenv global $(cat .ruby-version)
```

Comment:
Setting the global version of ruby is not necessary. If you are in the `io-app` directory and you have rbenv properly working/initialized, the `gem install ....` will use the right ruby version specified in the `.ruby-version` file.

Reply:
not in my experience , on macOS it used the global version of ruby that incompatible with one of the dependencies because it needs local compilation